scripts/ErrorReporter/errorreport.py

```diff
@@ -39,13 +56,15 @@ def __init__(self, parent=None, show_continue_terminate=False):
 
         self.icon.setPixmap(QtGui.QPixmap(":/crying_mantid.png"))
 
-        self.requestTextBrowser.anchorClicked.connect(QtGui.QDesktopServices.openUrl)
+        self.requestTextBrowser.anchorClicked.connect(InterfaceManager.showWebPage)
```

Comment:
This will throw an error when called. `showWebPage` isn't static so you'll need to call it on an instance of `InterfaceManager`, i.e. `self.interface_manager.showWebPage` or `InterfaceManager().showWebPage`